spanner/spec/spanner_samples_spec.rb

```diff
@@ -1516,7 +1516,7 @@ def capture
     end
 
     expect(captured_output).to match(
-      /Database #{database.database_id} restored from backup is \d+% optimized/
+      /Database \w+ restored from backup is \d+% optimized/
```

Comment:
I see that in the test output there are results that match this. But do we know if this is always true? Only because I think we clear the databases at the end of the tests so I'm not sure if there will always be a result in this list call.

Reply:
@skuruppu Got it. In this case i am thinking to pull message like. `List database operations with optimized database filter found #{jobs.length} jobs` in sample code and check with the `/List database operations with optimized database filter found \d+% jobs/`